coffea/nanoevents/transforms.py

```diff
@@ -125,7 +125,7 @@ def local2global(stack):
     index = stack.pop()
     index = index.mask[index >= 0] + target_offsets[:-1]
     index = index.mask[index < target_offsets[1:]]
-    out = numpy.array(awkward.flatten(awkward.fill_none(index, -1)))
+    out = numpy.array(awkward.flatten(awkward.fill_none(index, -1), axis=None))
```

Comment:
added the `axis=None` here to support indices that are already more than single-jagged

Reply:
I assume here the link target collection is still single-jagged? Otherwise I don't think this algorithm works properly.

Reply:
yes, all target collections i'm considering so far are single-jagged